tools/src/main/resources/com/fulcrumgenomics/str/vcf/StrGenotypeDuplexMolecules.R

```diff
@@ -0,0 +1,86 @@
+#################################################################################
+# The MIT License                                                               #
+# Copyright (c) 2017 Fulcrum Genomics LLC                                       #
+# Permission is hereby granted, free of charge, to any person obtaining a copy  #
+# of this software and associated documentation files (the "Software"), to deal #
+# in the Software without restriction, including without limitation the rights  #
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell     #
+# copies of the Software, and to permit persons to whom the Software is         #
+# furnished to do so, subject to the following conditions:                      #
+# The above copyright notice and this permission notice shall be included in    #
+# all copies or substantial portions of the Software.                           #
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR    #
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,      #
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE   #
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER        #
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, #
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN     #
+# THE SOFTWARE.                                                                 #
+#################################################################################
+
+#################################################################################
+# R script to generate QC plots VcfToGenotypes
+#################################################################################
+
+options(warn = -1) # Don't emit warnings, only errors
+library(ggplot2)
+library(scales)  # Require by ggplot
+
+# Standard colors
+fgblue  = "#2E9DD7"
+fggreen = "#155936"
+fgred   = "firebrick3"
+fgcolors = c(fgblue, fggreen, fgred)
+
+args       = commandArgs(trailingOnly=T)
+inputFile  = args[1]
+outputFile = args[2]
+
+data = read.table(inputFile, sep="\t", header=F)
+
+pdf(outputFile, width=11, height=8.5)
+
+for (i in 1:nrow(data)) {
+    row = data[i,]
+    chromosome  = row[1]
+    start       = row[2]
+    end         = row[3]
+    unitLength  = row[4]
+    refNumUnits = row[5]
+    strName     = row[[6]]
```

Comment:
So I don't get a list returned in some cases: https://stackoverflow.com/questions/1169456/the-difference-between-and-notations-for-accessing-the-elements-of-a-lis